src/main/scala/org/apache/spark/sql/snowflake/SnowflakeSparkUtils.scala

```diff
@@ -0,0 +1,49 @@
+package org.apache.spark.sql.snowflake
+
+import org.apache.spark.sql.execution.datasources.{LogicalRelation, SaveIntoDataSourceCommand}
+import org.apache.spark.sql.execution.datasources.jdbc.{JDBCRelation, JdbcRelationProvider}
+
+object SnowflakeSparkUtils {
```

Comment:
`org.apache.spark.sql.JDBCRelation` is package private. So introduce `org.apache.spark.sql.snowflake.SnowflakeSparkUtils` to generate the class name.